tools/ci/run_unit_tests.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -ex
 target="${1:?"Specify a makefile target"}"
+# Set to 1 to temporarily ignore warnings
+export PERL_TEST_WARNINGS_ONLY_REPORT_WARNINGS=0
```

Comment:
Did you test it? I can't seem to execute it [on your branch](https://github.com/perlpunk/openQA/actions/workflows/prove.yml?query=branch%3Atest-warnings). There's no **Run workflow** button 🤔

Reply:
I think that button only appears if the workflow is configured for the manual dispatch. I don't worry about this change though and approved it nevertheless.

Reply:
I forgot to add the link where I tested it with a warning: https://app.circleci.com/pipelines/github/perlpunk/openQA/306/workflows/da68462a-3df4-404d-9196-89b25e8c8db4/jobs/1851

Reply:
Yes, `workflow_dispatch` like in `prove.yml`. You're right, though, the intention is to use it with CircleCI specifically

Reply:
Regarding the button - I think there is no button visible for you because you can't trigger a workflow run in my repo, where you don't have any access. I can see the button.